product_docs/docs/livecompare/3/command_line_usage.mdx

```diff
@@ -2,16 +2,51 @@
 navTitle: Command line usage
 title: Command-line usage
 originalFilePath: command_line_usage.md
-
+deepToC: true
 ---
 
 ## Compare mode
 
-Copy any `/etc/livecompare/template*.ini` to use in your project and adjust as necessary. See [Settings](settings/).
+Copy any `/etc/livecompare/template*.ini` to use in your project and adjust as necessary. See [Settings](settings/) for details.
```

Comment:
```suggestion
```
We should remove this altogether as templates currently no longer live at `/etc/livecompare/template*` The following line `cp ...` should just say create a new .ini file